File: app/src/main/java/com/callmangement/model/reports/MonthReportModel.java
package com.callmangement.model.reports;

import com.callmangement.model.complaints.ModelComplaintList;

import java.util.List;

public class MonthReportModel {
    String date;
    List<ModelComplaintList> list;

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public List<ModelComplaintList> getList() {
        return list;
    }

    public void setList(List<ModelComplaintList> list) {
        this.list = list;
    }
}

File: app/src/main/java/com/callmangement/model/reports/MonthReportModel.kt
package com.callmangement.model.reports

import com.callmangement.model.complaints.ModelComplaintList

class MonthReportModel {
    @JvmField
    var date: String? = null
    @JvmField
    var list: MutableList<ModelComplaintList>? = null
}
